src/codecs.ts

```diff
@@ -923,27 +923,35 @@ export type Policy = t.TypeOf<typeof Policy>;
 export type PolicyC = t.UnionC<[t.LiteralC<'redactEmail'>, t.LiteralC<'log'>]>;
 
 /** the codec of a route enabled in an AI integration */
-export type EnabledRouteC = t.TypeC<{
+export type EnabledRouteC<T extends t.Mixed> = t.TypeC<{
   /** the name of the enabled route */
-  routeName: t.Mixed;
+  routeName: T;
```

Comment:
That's a good question. `io-ts` complains if you pass pure javascript types in this case.
<img width="946" alt="Screenshot 2023-08-10 at 2 57 10 PM" src="https://github.com/transcend-io/cli/assets/12059676/e823cd90-55aa-4568-9e40-d7704a2c230d">

These TypeC definitions expect io-ts custom types.

Reply:
What about the io-ts equivalent, like t.string or something? 😂 I'm so far from understanding what's happening here, that you can feel free to ignore me. I'm just trying to understand it a little better.

Reply:
Another great question! I remember trying something like that and I got an error. Let me try again.

Reply:
You're right that there is a io-ts equivalent 😄 it's t.StringC

Reply:
Awesome. Thanks for the explanation. So this does mean that if you tried to implement `routeName: 'my/dummy/route'`, it would recognize that that is not supported, correct?

Reply:
It would because once we actually instantiate T, we pass in something like `OpenAIRouteNameC`, insted of `t.Mixed`,  where 

```
 t.UnionC<
  [t.LiteralC<'/v1/chat/completions'>, t.LiteralC<'/v1/embeddings'>]
>;
```